frontend/dockerfile/dockerfile_test.go

```diff
@@ -5338,6 +5339,112 @@ COPY --from=base /env_foobar /
 	require.Contains(t, string(dt), "/foobar:")
 }
 
+func testNamedImageContextTimestamps(t *testing.T, sb integration.Sandbox) {
+	cdAddress := sb.ContainerdAddress()
+	if cdAddress == "" {
+		t.Skip("test requires containerd worker")
```

Comment:
You could just call `ResolveImageConfig()` to get the image config without using the containerd puller.

Reply:
Using the same trick as [here](https://github.com/moby/buildkit/commit/4d23e260f348eb634d5cbc40657b7c8167fa7f38#diff-57cdc3e29d0ef725a50897a4fbd1c4f76353429fa66d3a785dcbc17dcd7b6f31R2182-R2184)?

Can switch to using that, but it definitely feels like a bit of a workaround, I think the containerd implementation is clearer (though of course relies on containerd tests).

Reply:
Have used the `contentutil` package here, along with the `readImage` test helper to get the image info.